lib/internal/modules/cjs/helpers.js

```diff
@@ -41,10 +41,9 @@ function makeRequireFunction(mod, redirects) {
     const { resolve, reaction } = redirects;
     const id = mod.filename || mod.id;
     require = function require(path) {
-      let missing = true;
       const destination = resolve(path);
       if (destination === true) {
-        missing = false;
+        reaction(new ERR_MANIFEST_DEPENDENCY_MISSING(id, path));
```

Comment:
Isn't this doing the opposite now? Before, this was only being called when `missing === true`.

Reply:
updated